internal/pkg/manifest/rd_web_svc.go

```diff
@@ -134,6 +134,7 @@ func (s *RequestDrivenWebService) windowsCompatibility() error {
 	if s.InstanceConfig.Platform == nil {
 		return nil
 	}
+	// Error out if user added Windows as platform in manifest.
```

Comment:
For a follow-up PR: we should also error out I think if the arch specific is ARM for RDWS

Reply:
Did it here.